.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+omit = test/*
+[report]
+omit = test/*
```

Comment:
What does this omit bit do? Whats the reason for it?

Reply:
Stops the files containing the tests from showing up in the coverage report, just makes it look cleaner as coverage already counts tests as passing themselves

Reply:
Sweet